Cargo.toml

```diff
@@ -91,7 +91,8 @@ genmesh = "0.6"
 ron = "0.4"
 
 [build-dependencies]
-vergen = "2.0"
+failure = "0"
```

Comment:
0?

Reply:
ya, so `vergen` uses `failure` under the hood with that "version". It doesn't re-export the crate, but uses the `Fallible<T>` type as a return type (shorthand for `Result<T, failure::Error>`).

I could PR to `vergen` first to re-export `failure`, or use an `.expect("..")` in `build.rs` if you prefer

Reply:
I for one would just prefer an expect.

Reply:
Yeah, that seems to be common practice in build scripts.

Reply:
Yay, we need https://github.com/rust-lang-nursery/failure/pull/279 for a better error message